### PR TITLE
{lib}[GCCcore/10.3.0] Check v0.15.2, googletest v1.11.0, JsonCpp v1.9.4

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/6846ee7a57a265e156bac463cb3c9407 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/g/googletest/googletest-1.11.0-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+sources = ['release-%(version)s.tar.gz']
+checksums = ['b4870bf121ff7795ba20d20bcdd8627b8e088f2d1dab299a031c1034eddc93d5']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+]
+# build twice, once for static, once for shared libraries
+configopts = ['', ' -DBUILD_SHARED_LIBS=ON ']
+separate_build_dir = True
```
Comment (branfosj) [inline easybuild/easyconfigs/g/googletest/googletest-1.11.0-GCCcore-10.3.0.eb:21]:
```suggestion
```

This is the deafult for CMakeMake, so we should remove it.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/71276f07a09d566a106bff0c5790864d for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/027267ef6c561d52ddb85e4c25f14514 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0305u19a.bear.cluster - Linux RHEL 8.4, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/9b7b61ef40048d80024f9abfa882db3d for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**